testing/test_skipping.py

```diff
@@ -861,9 +861,25 @@ def test_hello():
                 pass
         """
         )
-        result = pytester.runpytest("-rs")
+        result = pytester.runpytest("-rs", "--strict-markers")
         result.stdout.fnmatch_lines(["*unconditional skip*", "*1 skipped*"])
 
+    def test_wrong_strict_usage(self, pytester: Pytester) -> None:
```

Comment:
I think you meant `test_wrong_skip_usage` here.

Reply:
Err, yes, totally - good catch!